brainvisa/toolboxes/spm/hierarchies/brainvisa-3.2.0/MRI_spm.py

```diff
@@ -515,11 +516,17 @@ def cat_tissue_outputs(tissue_prefix, tissue_class):
             '<subject>_{acquisition}_to_avg_<acquisition_sequence>_jacobian_determinant',
               SetType('Jacobian determinant'),
               SetWeakAttr('space', 'average'),
+            'jd_<subject>_{baseline}_{followup}',
```

Comment:
Is it related to WriteNormalised or it came from some DIAMOND modifications?

Reply:
eh eh nope, it shouldn't be there :), it is Diamond related. I am going to try to remove it from this PR.